<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPDF_MainWindow</class>
 <widget class="QMainWindow" name="EditPDF_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>683</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="PDF_EditGraphicView" name="imageFrame">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolBox" name="Edited_toolBox">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>300</width>
         <height>491</height>
        </rect>
       </property>
       <attribute name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/toolbar_icon/pic/editPDF.png</normaloff>:/toolbar_icon/pic/editPDF.png</iconset>
       </attribute>
       <attribute name="label">
        <string>添加...</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTabWidget" name="toolsTab">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="tabBarAutoHide">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="tab_marker">
           <attribute name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/toolbar_icon/pic/marker.png</normaloff>:/toolbar_icon/pic/marker.png</iconset>
           </attribute>
           <attribute name="title">
            <string>荧光笔</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QWidget" name="operatingWidgetBtns" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QPushButton" name="btn_addMarker">
                 <property name="minimumSize">
                  <size>
                   <width>70</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>添加荧光笔</string>
                 </property>
                 <property name="icon">
                  <iconset resource="icons.qrc">
                   <normaloff>:/toolbar_icon/pic/marker.png</normaloff>:/toolbar_icon/pic/marker.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="adjust_widget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLineEdit" name="lineEdit_curLength">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>长度</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSlider_currentWidth">
                 <property name="minimum">
                  <number>10</number>
                 </property>
                 <property name="maximum">
                  <number>1000</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_curHeight">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>宽度</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSliderCurrentHeight">
                 <property name="minimum">
                  <number>10</number>
                 </property>
                 <property name="maximum">
                  <number>1000</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="hozi_widget" native="true">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>70</height>
                  </size>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QTextBrowser" name="reminder_oftransparency">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="html">
                     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;透明度: &lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSlider" name="horizontalSlider_transparency">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="maximum">
                     <number>255</number>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_setColor">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>设置创建颜色</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_text">
           <attribute name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/toolbar_icon/pic/Text.png</normaloff>:/toolbar_icon/pic/Text.png</iconset>
           </attribute>
           <attribute name="title">
            <string>文字</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QWidget" name="widget" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QPushButton" name="btn_addText">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>添加文字框</string>
                 </property>
                 <property name="icon">
                  <iconset resource="icons.qrc">
                   <normaloff>:/toolbar_icon/pic/Text.png</normaloff>:/toolbar_icon/pic/Text.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="textEditPlace">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>100</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="adjust_widget_2" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QWidget" name="hozi_widget_2" native="true">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>70</height>
                  </size>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <widget class="QTextBrowser" name="reminder_oftransparency_text">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="html">
                     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;透明度: &lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSlider" name="horizontalSlider_transparency_text">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="maximum">
                     <number>255</number>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget_2" native="true">
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <item>
                   <widget class="QPushButton" name="btn_setColor_textbackground">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>50</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>设置背景颜色</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btn_setColor_text">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>50</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>设置字体颜色</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btn_setColor_textfont">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>50</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>设置字体</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <attribute name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/toolbar_icon/pic/resume.png</normaloff>:/toolbar_icon/pic/resume.png</iconset>
       </attribute>
       <attribute name="label">
        <string>撤销</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QPushButton" name="btn_resumeLastOperation">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>回退</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/toolbar_icon/pic/resume.png</normaloff>:/toolbar_icon/pic/resume.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_resumeLastResume">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>撤销上次回退</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/toolbar_icon/pic/resumeTheresume.png</normaloff>:/toolbar_icon/pic/resumeTheresume.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>300</width>
         <height>491</height>
        </rect>
       </property>
       <attribute name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/toolbar_icon/pic/cancelSelect.png</normaloff>:/toolbar_icon/pic/cancelSelect.png</iconset>
       </attribute>
       <attribute name="label">
        <string>删除</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QPushButton" name="rermove_selected">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>删除当前选中的标记</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/toolbar_icon/pic/cancelSelect.png</normaloff>:/toolbar_icon/pic/cancelSelect.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove_all">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>删除所有的标记</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/toolbar_icon/pic/cancelSelect.png</normaloff>:/toolbar_icon/pic/cancelSelect.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>300</width>
         <height>491</height>
        </rect>
       </property>
       <attribute name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/toolbar_icon/pic/savePDF.png</normaloff>:/toolbar_icon/pic/savePDF.png</iconset>
       </attribute>
       <attribute name="label">
        <string>保存</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QLabel" name="image_res_label">
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>320</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>无图片显示，尝试刷新一下！</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_refresh">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string>刷新</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/toolbar_icon/pic/refresh.png</normaloff>:/toolbar_icon/pic/refresh.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_save">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string>保存编辑</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/toolbar_icon/pic/savePDF.png</normaloff>:/toolbar_icon/pic/savePDF.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>创建</string>
    </property>
    <addaction name="action_createMarker"/>
    <addaction name="action_createText"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>撤销</string>
    </property>
    <addaction name="action_resume"/>
    <addaction name="action_resumeTheResume"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>保存</string>
    </property>
    <addaction name="action_save"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>删除</string>
    </property>
    <addaction name="action_deleteThis"/>
    <addaction name="action_deleteAll"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_createMarker">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar_icon/pic/marker.png</normaloff>:/toolbar_icon/pic/marker.png</iconset>
   </property>
   <property name="text">
    <string>荧光笔</string>
   </property>
  </action>
  <action name="action_createText">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar_icon/pic/Text.png</normaloff>:/toolbar_icon/pic/Text.png</iconset>
   </property>
   <property name="text">
    <string>文本框</string>
   </property>
  </action>
  <action name="action_resume">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar_icon/pic/resume.png</normaloff>:/toolbar_icon/pic/resume.png</iconset>
   </property>
   <property name="text">
    <string>撤销上一步</string>
   </property>
  </action>
  <action name="action_resumeTheResume">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar_icon/pic/resumeTheresume.png</normaloff>:/toolbar_icon/pic/resumeTheresume.png</iconset>
   </property>
   <property name="text">
    <string>恢复上一步</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar_icon/pic/savePDF.png</normaloff>:/toolbar_icon/pic/savePDF.png</iconset>
   </property>
   <property name="text">
    <string>保存编辑</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar_icon/pic/quit.png</normaloff>:/toolbar_icon/pic/quit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_deleteThis">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar_icon/pic/cancelSelect.png</normaloff>:/toolbar_icon/pic/cancelSelect.png</iconset>
   </property>
   <property name="text">
    <string>删除选中</string>
   </property>
  </action>
  <action name="action_deleteAll">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar_icon/pic/cancelSelect.png</normaloff>:/toolbar_icon/pic/cancelSelect.png</iconset>
   </property>
   <property name="text">
    <string>删除全部</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PDF_EditGraphicView</class>
   <extends>QGraphicsView</extends>
   <header location="global">pdf_editgraphicview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
